app/controllers/miq_policy_controller.rb

```diff
@@ -1123,5 +1126,18 @@ def features
     end
   end
 
+  def breadcrumbs_options
+    breadcrumbs = [{:title => _("Control")}, {:title => _("Explorer")}] if action_name == "explorer"
+    breadcrumbs = [{:title => _("Control")}, {:title => _("Simulation")}] if action_name == "rsop"
+    breadcrumbs ||= [{:title => _("Control")}]
+    {
+      :breadcrumbs => breadcrumbs,
+    }
```

Comment:
This is not very readable, ping me about it or try to use some of the techniques form my comments above.

Reply:
What about
```
{
  :breadcrumbs => [{:title => _("Control")}] + case action_name
                                               when "rsop"     then [{:title => _("Simulation")}]
                                               when "explorer" then [{:title => _("Explorer")}]
                                               else []
                                               end
}
```